package.json

```diff
@@ -9,7 +9,7 @@
     "hof guide"
   ],
   "engines": {
-    "node": "^16.15.0"
+    "node": "^15.14.0"
```

Comment:
If you are happy with changing the node version - the some of the guidance will need to be updated (in the readme)